prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -14,13 +14,29 @@
                       (navigateEnrollee)="onNavigateEnrollee($event)"
                       (sendBulkEmail)="onSendBulkEmail()"></app-enrollee-table>
 
-  <div class="d-flex justify-content-between align-items-center mt-4">
+  <div class="d-flex justify-content-between align-items-center mt-4 mb-4">
     <span>
+      <app-alert *ngIf="!enrollee?.linkedEnrolleeId && enrollee?.possiblePaperEnrolmentMatch"
+                 type="warning">
+        <ng-container #alertTitle
+                      class="alert-title">
+          Possible paper enrolment matches found
+        </ng-container>
+        <ng-container #alertContent
+                      class="alert-content">
+          Use the Enrollee search to determine which paper enrolments match
+        </ng-container>
+      </app-alert>
       <div *ngIf="enrolment?.enrollee.gpid"
-            class="mb-4 align-self-center">
+           class="mb-4 align-self-center">
           <strong>GPID:</strong> {{ enrolment?.enrollee.gpid }}
       </div>
-
+      <div *ngIf="enrollee?.linkedEnrolleeId">
+        <a [routerLink]=""
+           (click)="onRoute([enrollee?.linkedEnrolleeId, AdjudicationRoutes.ENROLLEE_OVERVIEW])">
+          {{ (enrolment?.enrollee.gpid | contains: paperEnrolleeGpidFilter) ? 'Linked' : 'Paper' }} Enrolment link
```

Comment:
The word link here is self explanatory, best to remove it. If then it's not to obvious what clicking the link does you can use a verb in front of it like "Go to linked enrolment" "Go to paper enrolment"